### ISSUE TITLE
The font used for HTML embeds is ugly

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-ios#265]
The font used for HTML embeds is ugly. Can we use the system font ?

**Comment by teolemon** [comment]
https://stackoverflow.com/questions/12533445/how-to-change-uiwebview-default-font
``` func webViewDidFinishLoad(_ webView: UIWebView) {
    webView.stringByEvaluatingJavaScript(from: "document.getElementsByTagName('body')[0].style.fontFamily =\"-apple-system\"")
}
``` 
Swift 3 solution with system font

**Comment by codethebeard** [comment]
I can take this on if ya'll would like me to. :)

**Comment by teolemon** [comment]
@codethebeard it would be awesome :-) 
If you can blend it beautifully using the right fonts (the default system one ?)

**Comment by codethebeard** [comment]
I'll try my best. Thanks!

**Comment by codethebeard** [comment]
So, I'm having a lot of trouble getting the project to launch without crashing. I've walked through the setup steps with Carthage and when the app opens it immediately crashes due to a missing Library. Might be related to this: https://github.com/openfoodfacts/openfoodfacts-ios/issues/306 maybe?

**Comment by SuzGupta** [comment]
@codethebeard I was new to Carthage and could not initially get the project to build. It helped me to use `update` rather than `bootstrap`. Apologies if I am saying things you already know, but I found https://www.chrisjmendez.com/2016/10/30/carthage-cheat-sheet/ helpful, as well as this Ray Wenderlich Carthage tutorial: https://www.raywenderlich.com/416-carthage-tutorial-getting-started

**Comment by codethebeard** [comment]
@SuzGupta Thanks for the info. I haven't used Carthage before (mostly a CocoaPods kinda guy). I'll try giving this a shot and post back if it helps. There were a few libraries that were missing apparently.

**Comment by codethebeard** [comment]
Seems like the above Carthage cheat sheet helped a bunch. Thanks @SuzGupta

**Comment by SuzGupta** [comment]
You're welcome—I'm glad it helped! I look forward to us getting an improved version out in the App Store... I will then start scanning my whole pantry and get to know this whole effort better :)

**Comment by codethebeard** [comment]
Resolved in #375
